sample/development/README.md

```diff
@@ -0,0 +1,29 @@
+# About
+
+This Roku channel is built for SDK development.
+
+## Initializing Project
+
+To test with this Roku channel, create a `test_config.json` file under the [source](./source) directory and add the edge configuration as shown below.
+
+```json
+{"config_id":"your edge config id"}
+```
+
+From the sample/development directory run the following command in a terminal window to link the development app to the SDK:
+```
+make link-sdk
+```
+
+### VSCode plugin
+
+For developers using the [VSCode plugin (BrightScript Language extension for VSCode)](https://marketplace.visualstudio.com/items?itemName=RokuCommunity.brightscript), the `launch.json` under `.vscode` is configued to work with the plugin.
+
+To deploy the Roku channel to your Roku device, you need to provide a `settings.json` file under the `.vscode` directory and add the below information.
```

Comment:
nit: can we add screenshot of the two files from our IDE
<img width="895" alt="Screenshot 2023-08-03 at 9 52 24 AM" src="https://github.com/adobe/aepsdk-roku/assets/4697559/e8756e45-c63b-4425-be4b-0e82f5d8483b">

Reply:
No, we shouldn't do it.  I intend to ignore the settings.json to avoid checking in that info.